builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -0,0 +1,189 @@
+    echo ""
+    echo " Please install a container runtime and run this install again."
+    echo ""
+    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+
+    exit 1
+fi
+
+# Create iotedge group
+if ! /usr/bin/getent group iotedge >/dev/null; then
+    %{_sbindir}/groupadd -r %{iotedge_group}
+fi
+
+# Create iotedge user
+if ! /usr/bin/getent passwd iotedge >/dev/null; then
+    %{_sbindir}/useradd -r -g %{iotedge_group} -c "iotedge user" -s /bin/nologin -d %{iotedge_home} %{iotedge_user}
+fi
+
+# Add iotedge user to moby-engine group
+if /usr/bin/getent group docker >/dev/null; then
+    %{_sbindir}/usermod -a -G docker %{iotedge_user}
+fi
+
+# Add iotedge user to aziot-identity-service groups
+if /usr/bin/getent group aziotcs >/dev/null; then
+    %{_sbindir}/usermod -aG aziotcs %{iotedge_user}
+fi
+if /usr/bin/getent group aziotks >/dev/null; then
+    %{_sbindir}/usermod -aG aziotks %{iotedge_user}
+fi
+if /usr/bin/getent group aziotid >/dev/null; then
+    %{_sbindir}/usermod -aG aziotid %{iotedge_user}
+fi
+exit 0
+
+%post
+if [ ! -f '/etc/aziot/config.toml' ]; then
+    echo "==============================================================================="
+    echo ""
+    echo "                              Azure IoT Edge"
+    echo ""
+    echo "  IMPORTANT: Please configure the device with provisioning information."
+    echo ""
+
+    if [ -f '/etc/iotedge/config.yaml' ]; then
+        echo "  Detected /etc/iotedge/config.yaml from a previously installed version of IoT Edge."
+        echo "  You can import the previous configuration using:"
+        echo ""
+        echo "    iotedge config import"
+        echo ""
+        echo "Alternatively, copy the configuration file at /etc/aziot/config.toml.edge.template to /etc/aziot/config.toml,"
+    else
+        echo "Copy the configuration file at /etc/aziot/config.toml.edge.template to /etc/aziot/config.toml,"
+    fi
+
+    echo "  update it with your device information, then apply your configuration changes with:"
+    echo ""
+    echo "    iotedge config apply"
+    echo ""
+    echo "  You may need to run iotedge config commands with sudo, depending on your environment."
+    echo ""
+    echo "==============================================================================="
+fi
+%systemd_post aziot-edged.service
+
+%preun
+%systemd_preun aziot-edged.service
+
+%postun
+%systemd_postun_with_restart aziot-edged.service
+
+%files
+%defattr(-, root, root, -)
+
+# bins
+%{_bindir}/iotedge
+%{_libexecdir}/aziot/aziot-edged
+
+# config
+%attr(600, root, root) %{aziot_confdir}/config.toml.edge.template
+%attr(400, %{iotedge_user}, %{iotedge_group}) %{iotedge_confdir}/config.toml.default
+%attr(700, %{iotedge_user}, %{iotedge_group}) %dir %{iotedge_confdir}/config.d
+%config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
+
+# man
+%{_mandir}/man1/iotedge.1.gz
+%{_mandir}/man8/aziot-edged.8.gz
+
+# systemd
+%{_unitdir}/aziot-edged.service
+
```

Comment:
Please add the systemd config
`%{_presetdir}/00-aziot-edged.preset`

Reply:
I believe this is a CentOs required file. do you know if we would still need that for mariner?

Reply:
It's a template for config file for IotEdge. we should have it.